packages/wallet/src/redux/outbox/state.ts

```diff
@@ -11,7 +12,7 @@ export interface QueuedTransaction {
   processId: string;
 }
 export type DisplayOutbox = DisplayAction[];
-export type MessageOutbox = WalletEvent[];
+export type MessageOutbox = Array<WalletEvent | JsonRpcResponseAction>;
```

Comment:
What's the plan for dealing with these two: `WalletEvent` and `JsonRpcResponseAction`? Will there always be these two fundamental different types of outbound messages?

Reply:
No, `WalletEvent`s will eventually be superseded by `JsonRpcResponseAction`s as we shift towards the json rpc API.  For now this is in place to not break our existing reducers.